src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayOfRent = 40;
```

Comment:
The name of the variable 'dayOfRent' is not meaningful at all. In the future try to use some simple and meaningful names. In this case, it would be more understandable if you would call it 'general price'.

But it's a good practice declaring variables for reusable data 👍